.travisci/pub_html.sh

```diff
@@ -30,6 +30,11 @@ else
     exit 0
 fi
 
+if [ `git rev-list HEAD...origin/$TRAVIS_BRANCH --count` != 0 ]; then
```

Comment:
Oh, nice. I forgot about `--count` option. This is so helpful for one of my scripts.

Reply:
The `--left-right` flag is really useful as well.